### DISCUSSION TITLE
Screen goes black when alt + tab a game or when changing brightness

### DISCUSSION THREAD

**Discussion by ophilbert** [MonitorControl/MonitorControl#1641] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Screen goes entirely black randomly when I alt + tab a game or leave a game.
It doesn't happen all the times.

Sometimes it does that when changing the brightness too.

![Capture d’écran 2023-10-13 à 11 02 20](https://github.com/MonitorControl/MonitorControl/assets/1815629/d166c479-9056-4cb2-9802-f206253f03f0)

This only happen to my main screen which is an old Apple Cinema Display.

The only way to get it back is either go to Mac OS Settings > Monitors > Organize display and move the screens layouts a bit so the screen lights up again. I found that quitting Monitor Control also did the trick so I believe it could come from here or quitting the app forces a refresh of something.

### Steps to reproduce

Unfortunately I don't have any steps to reproduce as sometimes it works perfectly and sometimes the screen goes black.

### Expected behavior

The screen should never stay black.

### Anything else?

Happy to help pinning down this issue.
Maybe there's something in the logs files but didn't find anything in Application Support.

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 13.2.1
- Mac model: Mackbook Pro M1 15"
- MonitorControl version: 4.2.0
- Monitor(s): Macbook M1, Apple Cinema Display 27, Samsung Odyssey G3
- Apple Silicon/M1 (yes or no): yes
- Cinema display is connected to my M1 using 2 adapters. Thunderbolt to usb > usb to usb C
```

**Comment by ophilbert** [comment]
@waydabber is there a log file or something I can look into to help you on this matter ?

**Reply by waydabber** [reply]
Not really, no. Honestly I have no idea what could this be. Did you try BetterDisplay - does it produce the same issue? With BetterDisplay you can see detailed logging if you start the app from Terminal, adding the `-v` parameter:

`/Applications/BetterDisplay.app/Contents/MacOS/BetterDisplay -v`

**Reply by ophilbert** [reply]
Gonna try BetterDisplay and get back to you.
If this works i will close this issue.
Otherwise i will close this issue and make an issue on BetterDisplay with the log infos.
Thanks.

**Reply by waydabber** [reply]
Thank you! Also could you observe how the black screen looks like? Does the panel backlight also go away? Does this look like some kind of modeswitch or a software induced blackout? In MonitorControl you can try turing off DDC or gamma table manipulation under Settings/Displays and see if the issue still happens.
